support-frontend/Brewfile.lock.json

```diff
@@ -19,28 +19,28 @@
     },
     "tap": {
       "guardian/devtools": {
-        "revision": "de424b3f5c336b4f1943e73f31bd89a2b22076e0"
+        "revision": "70411dc711b940876431943f0b1b6dd6c978a382"
```

Comment:
I think we normally don't commit this file?

Reply:
It's not included in the `.gitignore` file, so I assumed it was to be left in 😕.

@tomrf1 can you advise?

Reply:
honestly I don't know why we have this file, I'd be fine with deleting it and adding to the .gitignore but it's not essential now!

Reply:
Ok, I've removed it and added it to .gitignore

Reply:
thanks!